applications/GeoMechanicsApplication/tests/flow_rate_heterogeneous_soil.gid/ProjectParameters.json

```diff
@@ -61,7 +60,6 @@
         "min_alpha":                          0.1,
         "max_alpha":                          2.0,
         "line_search_tolerance":              0.5,
```

Comment:
Same here, deletion is correct, , the defaults in geomechanics_U_Pw_solver.py and in geomechanics_solver.py are both false.
( In future we probably should make sure there is only 1 place where this setting has its default. )